parachains/runtimes/starters/shell/src/lib.rs

```diff
@@ -69,10 +69,10 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("shell"),
 	impl_name: create_runtime_str!("shell"),
 	authoring_version: 1,
-	spec_version: 2,
+	spec_version: 9270,
 	impl_version: 0,
 	apis: RUNTIME_API_VERSIONS,
-	transaction_version: 1,
+	transaction_version: 7,
```

Comment:
```suggestion
	transaction_version: 1,
```
Do not bump `transaction_version`